drivers/input/input_sbus.c

```diff
@@ -129,6 +134,15 @@ static void input_sbus_input_report_thread(const struct device *dev, void *dummy
 	int ret;
 	bool connected_reported = false;
 
+	/* Before officially starting the report, let's first report a disconnection. */
+	if(cfg->connection_code != INPUT_KEY_RESERVED){
```

Comment:
Please adhere to the code style as in other parts of the file

```suggestion
	if (cfg->connection_code != INPUT_KEY_RESERVED) {
```